src/metrecs/radio_utils.py

```diff
@@ -0,0 +1,166 @@
+from typing import Dict, List
+from numpy.typing import ArrayLike
+from scipy.spatial.distance import pdist, squareform
+from scipy.spatial import distance
+from scipy.stats import entropy
+
+import numpy as np
+import math
+
+def harmonic_number(n: int) -> float:
+    """Returns an approximate value of n-th harmonic number.
+    The harmonic number can be approximated using the first few terms of the Taylor series expansion:
+    Source: http://en.wikipedia.org/wiki/Harmonic_number
+    """
+    # Euler-Mascheroni constant
+    gamma = 0.57721566490153286060651209008240243104215933593992
+    return gamma + math.log(n) + 0.5 / n - 1.0 / (12 * n**2) + 1.0 / (120 * n**4)
+
+def mrr(n):
+  """
+  return a list with n mean reciprocal rank weights for ranks from 1 to n
+  """ 
+  return list([1 / rank / harmonic_number(n) for rank in range(1, n + 1)])  
```

Comment:
Is this meant to be 'mean reciprocal rank'. Not sure if name is appropriate.

Reply:
It is the same function as 'normalized_scaled_harmonic_number_series' in 'utils.py'. For my understanding we are not computing the the Mean Reciprocal Rank score, rather these are the positional weights, or am I missing something?